offline/QA/Jet/RhosinEvent.cc

```diff
@@ -138,23 +140,61 @@ int RhosinEvent::process_event(PHCompositeNode* topNode)
     TowerRho* towerrho = findNode::getClass<TowerRhov1>(topNode, m_area_rho_node);
     if (!towerrho)
     {
-      std::cout << "RhosinEvent::process_event - Error can not find towerrho " << m_area_rho_node << std::endl;
+      std::cout << "RhosinEvent::process_event - Warning can not find towerrho " << m_area_rho_node << std::endl;
+      //exit(-1);
+    }
+    else{
+      h1_area_rho->Fill(towerrho->get_rho());
+      h1_area_rho_sigma->Fill(towerrho->get_sigma());
+    }
+
+    EventRho* eventrho = findNode::getClass<EventRhov1>(topNode, m_area_rho_node);
+    if (!eventrho)
+    {
+      std::cout << "RhosinEvent::process_event - Warning can not find eventrho " << m_area_rho_node << std::endl;
+      //exit(-1);
+    }
+    else{
```

Comment:
Minor suggestion in line with above:
```suggestion
    else
    {
```